pkg/users/account-details.js

```diff
@@ -101,12 +101,23 @@ export function AccountDetails({ accounts, groups, current_user, user, shells })
     useEffect(() => {
         get_expire(user).then(setExpiration);
 
-        // Watch `/var/run/utmp` to register when user logs in or out
-        const handle = cockpit.file("/var/run/utmp", { superuser: "try", binary: true });
-        handle.watch(() => {
-            get_expire(user).then(setExpiration);
-        }, { read: false });
-        return handle.close;
+        const lastlog_version = cockpit.spawn(["test", "-e", "/var/run/utmp"], { binary: true, err: true });
```

Comment:
This doesn't change between users, so please do this only once during page initialization, and pass the path as a property. Also, this doesn't need to be `binary`, it only cares about the success or not. `err: true` is also [invalid](https://cockpit-project.org/guide/latest/cockpit-spawn.html), please use `err: "ignore"`.